.changeset/cruel-clubs-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@tanstack/react-form': major
```

Comment:
Just checking if this should be major?

Reply:
Right, I did that quickly but it's not major, I'll update it!